TI_Practice4/Pactice4.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace TI_Practice4
+{
+    class Practice4
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please write something.");
+            String input = Console.ReadLine();
+
+            Palindrome palindrome1 = new Palindrome();
```

Comment:
1. why do you need 2 variables for this?
2. these are basically not palindromes, but just a word that can be a palindrome and can be not. I'd suggest renaming the type to PalindromeValidator or PalindromeChecker